src/node_jsvmapi.h

```diff
@@ -288,13 +321,17 @@ NODE_EXTERN bool napi_is_exception_pending(napi_env e);
 NODE_EXTERN napi_value napi_get_and_clear_last_exception(napi_env e);
```

Comment:
Yes, I knew about these and I skipped them so we could talk about whether or not we should instrument them to return napi_status. Went ahead and did instrument them after our meeting Friday. Sounds like the right thing to do.